napari/_qt/qt_dims_slider.py

```diff
@@ -223,33 +227,81 @@ def _update_slice_labels(self):
 
     @property
     def fps(self):
+        """Frames per second for animation."""
         return self._fps
 
     @fps.setter
     def fps(self, value):
+        """Frames per second for animation.
+
+        Paramters
+        ---------
+        value : float
+            Frames per second for animation.
+        """
         self._fps = value
         self.play_button.fpsspin.setValue(abs(value))
         self.play_button.reverse_check.setChecked(value < 0)
         self.fps_changed.emit(value)
 
     @property
     def loop_mode(self):
+        """Loop mode for animation.
+
+        Available options for the loop mode string enumeration are:
+        - LoopMode.ONCE
```

Comment:
What do we think, it this too likely to become out of date easily if changes are made to `napari._qt._constants.LoopMode`? I could just refer to the docstring at `napari._qt._constants.LoopMode` instead of repeating this information.

Reply:
I would say there's equal risk that any of these docstrings go out of date with changes :)
I don't expect there to be any more loop modes added (any time soon) so I think repeating that info is fine here.  You can make the call